### PR TITLE
React Native 5.0.0 - release notes

### PR DESCRIPTION
**Pull request by yangyansong-adbe**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html).
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Review by cacheung — APPROVED** [review]

File: src/pages/documentation/release-notes/index.md [hunk 1]
```diff
@@ -4,6 +4,24 @@ description: Release notes and change logs for the Adobe Experience Platform Mob
 
 # Release notes
 
+## June 20, 2023
+
+## AEP React Native libraries 5.0.0
+
+The following React Native libraries have been upgraded to use the Adobe iOS SDK 4.x. Starting with version 5.0.0 of the Adobe React Native SDK, all React Native libraries that share the same major version are compatible with each other.
```
Comment (calebk1) [inline src/pages/documentation/release-notes/index.md:11]: Is the Adobe iOS SDK 4.x the right name?

**Review by calebk1 — COMMENTED** [review]
Just a question!

File: src/pages/documentation/release-notes/index.md (hunk 1 above)
Comment (emdobrin) [inline src/pages/documentation/release-notes/index.md:11]: The following React Native libraries have been upgraded to use the version 4 of the Adobe Experience Platform Mobile SDK for iOS. 
Starting with this release, all React Native libraries that share the same major version are compatible with each other.

File: src/pages/documentation/release-notes/index.md [hunk 2]
```diff
@@ -4,6 +4,24 @@ description: Release notes and change logs for the Adobe Experience Platform Mob
...
+
+ - [@adobe/react-native-aepcore => 5.0.0](https://www.npmjs.com/package/@adobe/react-native-aepcore/v/5.0.0)
```
Comment (emdobrin) [inline src/pages/documentation/release-notes/index.md:13]: for consistency with the previous notes, let's use the `libname:5.0.0` notation, e.g: `@adobe/react-native-aepcore:5.0.0`

File: src/pages/documentation/release-notes/index.md [hunk 3]
```diff
@@ -4,6 +4,24 @@ description: Release notes and change logs for the Adobe Experience Platform Mob
 
 # Release notes
 
+## June 20, 2023
+
+## AEP React Native libraries 5.0.0
```
Comment (emdobrin) [inline src/pages/documentation/release-notes/index.md:9]: h3

**Comment by yangyansong-adbe** [comment]
@emdobrin @calebk1 Thank you for the review comments. They are all addressed now.

**Review by emdobrin — APPROVED** [review]

**Pull request merged**